src/DetectChanges/BCBreak/ClassBased/AncestorRemoved.php

```diff
@@ -21,20 +20,20 @@ final class AncestorRemoved implements ClassBased
 {
     public function __invoke(ReflectionClass $fromClass, ReflectionClass $toClass): Changes
     {
-        $removedAncestors = array_merge(
```

Comment:
Similar to above: would suggest keeping `array_merge()` and `array_diff()` here, since they are well known engine constructs that are as-safe-as PSL here?

Reply:
As per discussion with https://github.com/Roave/BackwardCompatibilityCheck/pull/306#discussion_r635004153, will roll with the `Psl\` version for now, as it is seems like I underestimated just how bad internal PHP functions are :|